src/interfaces/alteration-interfaces/business-snapshot-interface.ts

```diff
@@ -7,10 +7,10 @@ export interface EntitySnapshotIF {
   authInfo: AuthInformationIF
   orgPersons: OrgPersonIF[]
   addresses: AddressesIF
+  resolutions?: ResolutionsIF[]
```

Comment:
Why move this? IMO it's not "tombstone"  entity info data like the others (esp since optional). I recommend moving it back, or perhaps to its own block/comment below?

Reply:
Wanted to move it away from the comment: 
"Properties exclusive to Companies"

I didn't think a co-op was a company for some reason?

Reply:
Coop isn't a corp, but it is a business (which I think is a company).

Move to its own block below then.

Reply:
I'm just worried there are other resolutions other than special resolutions?

Reply:
Perfect 👍